pom.xml

```diff
@@ -100,7 +100,6 @@
     <module>processing</module>
     <module>hadoop</module>
     <module>integration/spark-common</module>
-    <module>integration/spark-datasource</module>
```

Comment:
Can you describe in PR content why do you need remove this and add it to spark-2.1,2.2,2.3 profile?

Reply:
Remoing module and adding to spark 2.1,2.2,2.3 is adaption to switching profile performance. This module's pom.xml profiles corresponds to the profile of spark version.

Reply:
please add it to PR content